lib/cjs/generated/rum.d.ts

```diff
@@ -151,6 +151,50 @@ export declare type RumActionEvent = CommonProperties & ViewContainerSchema & {
     };
     [k: string]: unknown;
 };
+/**
+ * Schema of all properties of an Transition event
+ */
+export declare type RumTransitionEvent = CommonProperties & StreamContainerSchema & {
+    /**
+     * RUM event type
+     */
+    readonly type: 'transition';
+    /**
+     * Transition properties
+     */
+    readonly transition: {
+        /**
+         * Type of the transition
+         */
+        readonly type: string;
+        /**
+         * UUID of the transition
+         */
+        readonly id?: string;
+        /**
+         * Timestamp in the player
+         */
+        readonly ts?: number;
+        /**
+         * Buffer starvation duration, the amount of time spent rebuffering
+         */
+        readonly bsd?: number;
+        /**
+         * Media start delay, the amount of time spent loading before playing
+         */
+        readonly msd?: number;
+        /**
+         * Error code, as reported by the player
+         */
+        readonly ec?: number;
```

Comment:
Thinking about how the data will be queried, it may be easier to understand if we avoid using abbreviations. WDYT?